src/features/wallet/root/ActivityDetails/HotspotTransaction.tsx

```diff
@@ -14,29 +14,58 @@ import Text from '../../../../components/Text'
 import PaymentItem from './PaymentItem'
 import { reverseGeocode } from '../../../../utils/location'
 
+const isAssert = (
+  arg: AnyTransaction | PendingTransaction,
+): arg is AssertLocationV1 => 'lat' in arg
+const isGateway = (
+  arg: AnyTransaction | PendingTransaction,
+): arg is AddGatewayV1 => 'gateway' in arg
+const isTransfer = (
+  arg: AnyTransaction | PendingTransaction,
+): arg is TransferHotspotV1 => 'seller' in arg
```

Comment:
Hmm this kinda sucks there isn't a better way to tell which transaction is which. Maybe we should check for `amountToSeller` as thats pretty specific to transfers. Would hate for some other transaction to have `seller` in it

Reply:
Actually now im thinking maybe seller is better, because `amountToSeller` is also optional. So ill leave it